includes/Story_Post_Type.php

```diff
@@ -132,6 +140,21 @@ public function register() {
 			]
 		);
 
+		$active_publisher_logo_id = absint( get_option( Settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO ) );
+
+		register_post_meta(
```

Comment:
I think this code could live in the media namespace. Similar to muting 

https://github.com/google/web-stories-wp/blob/main/includes/Media/Video/Muting.php

Reply:
That namespace doesn't seem fitting at all. The `Media` namespace contains all sorts of code that for media items / attachments: video muting, image sizes, SVG upload, etc.

This code, on the other hand, simply registers some post meta for the story post type holding an integer which happens to be an attachment ID.